src/Features/Core/Portable/CodeFixes/FixAllOccurrences/FixMultipleCodeAction.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.CodeFixes
+{
+    /// <summary>
+    /// Fix multiple occurrences code action.
+    /// </summary>
+    internal partial class FixMultipleCodeAction : FixAllCodeAction
+    {
+        private readonly string _title;
+        private readonly string _previewChangesDialogTitle;
+        private readonly string _computingFixWaitDialogMessage;
+
+        internal FixMultipleCodeAction(FixMultipleContext fixMultipleContext, FixAllProvider fixAllProvider, string title, string previewChangesDialogTitle, string computingFixWaitDialogMessage)
```

Comment:
I am not sure on this FixMultipleContext concept. wondering why it is not either collection of code actions or a variation of FixAllProvider.

don't we already have FixAllContext?